CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2021-09-30
+
+This release is rolling out to all regions - estimated time for completed roll out is 2021-10-07 for public cloud and 2021-10-14 for sovereign clouds.
+
+### Release Notes
+
+* Preview Features
+  * Enable multiple service account issuers for clusters using version >= 1.22, also fixing CNCF Validation issues.
```

Comment:
This is preview just because 1.22 is preview. We should call that out since we won't be separating this feature per se.

Reply:
1.22 and Preview is already called out?

Reply:
I saying we should call out this is preview _only_ because 1.22 is preview. And will GA with 1.22 GA automatically.